packages/create/templates/hello-drizzle/drizzle/0000_hello.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE IF NOT EXISTS "dbos_hello" (
```

Comment:
Drizzle automatically makes this and is the system that only does forward migrations, correct?

Reply:
Yeah